libraries/ts-command-line/src/providers/CommandLineParameterProvider.ts

```diff
@@ -668,6 +684,9 @@ export abstract class CommandLineParameterProvider {
     for (const parameter of this._parameters) {
       const value: unknown = data[parameter._parserKey!];
       parameter._setValue(value);
+      if (parameter._postParseValidation) {
+        parameter._postParseValidation(parameter._getValue());
+      }
```

Comment:
Can we `parameter._postParseValidation?.(parameter._getValue())` here to match the preparse styling?